registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/mapper/PacketInfoMapper.java

```diff
@@ -99,16 +99,32 @@ public static RegAbisRefEntity convertRegAbisRefToEntity(RegAbisRefDto regAbisRe
 	 *            the language
 	 * @return the json values
 	 */
+
 	private static String getJsonValues(JsonValue[] jsonNode, String language) {

```

Comment:
if this method is not used, can we remove it, since it is only private method ?

Reply:
Done